.changeset/dry-falcons-rest.md

```diff
@@ -0,0 +1,6 @@
+---
+'@crowdstrike/glide-core': patch
+---
+
+- Dropdown and Radio Group no longer reset their `value` after initial render when `value` is set initially.
```

Comment:
We have someone who'll be running into this bug soon. So we should probably do a release as soon as we can.